packages/rolldown/src/utils/validator.ts

```diff
@@ -862,5 +865,10 @@ export function getOutputCliKeys(): string[] {
 }
 
 export function getJsonSchema(): ObjectSchema {
-  return toJsonSchema(CliOptionsSchema) as ObjectSchema;
+  return toJsonSchema(CliOptionsSchema, {
+    // errorMode: 'ignore' is set to ignore `never` schema
+    // there's no way to suppress the error one-by-one
+    // https://github.com/fabian-hiller/valibot/issues/1062
+    errorMode: 'ignore',
+  }) as ObjectSchema;
```

Comment:
I think we need a unit test to ensure that `manualChunks` is the one thing ignored here. But I didn't find a place to write that.